docs-src/product/release-stages/public-preview.md

```diff
@@ -0,0 +1,22 @@
+---
+id: public-preview
+title: Public Preview
+description: Learn more about the Public Preview release stage
+sidebar_label: Public Preview
+tags:
+  - product release stages
+  - term
+---
+
+**Access:** Public preview of New Features are available to everyone.
+
+**Classification:** Public preview of New Features may be further developed and tested.
+These New Features are undergoing further refinement and are recommended for production usage.
+There may be exceptions for different features, this is the typical expectation.
+Any variation will be covered in documentation.
+
+**Feedback:** Temporal encourages Temporal Cloud users to submit feedback directly through the [Temporal Support Team](/cloud/support-create-ticket).
+This feedback will assist in guiding the improvements for General Availability.
+Temporal self-hosted users are invited to share feedback via the Community Slack channel; by reaching out directly to the Temporal Product team at product@temporal.io, or by logging issues on the appropriate [GitHub repository](https://github.com/temporalio).
```

Comment:
```suggestion
**Feedback:** Temporal users are invited to share feedback via the [Community Slack](http://t.mp/slack), by reaching out directly to the Temporal Product team at product@temporal.io, or by creating issues in the relevant [GitHub repository](https://github.com/temporalio).
Temporal encourages Temporal Cloud users to submit feedback [via support ticket](/cloud/support-create-ticket).
This feedback will assist in guiding the improvements for General Availability.
```

- oss first
- via support team feels less specific than "open a ticket", eg they're on the #support channel.